app/assets/stylesheets/mobile/modal.scss

```diff
@@ -68,7 +68,6 @@
   button {
     margin-top: 10px;
     display: block;
```

Comment:
I could be wrong, but the other two CSS rules for these buttons don't seem to be useful either (`margin-top: 10px; display: block).